site/content/docs/0.0/utilities/spacing.md

```diff
@@ -1,11 +1,362 @@
 ---
 layout: docs
 title: Spacing
-description: OUDS Web includes a wide range of shorthand responsive margin, padding and gap utility classes to modify an element's appearance.
+description: OUDS Web includes a wide range of shorthand responsive margin, padding, and gap utility classes to modify an element’s appearance.
 group: utilities
 aliases:
   - "/docs/utilities/spacing/"
 toc: true
 ---
 
-{{< callout-soon "page" >}}
+## Margin and padding
+
+Assign responsive-friendly `margin` or `padding` values to an element or a subset of its sides with shorthand classes. Includes support for individual properties, all properties, and vertical and horizontal properties. Classes are built from a default Sass map ranging from `none` to `jumbo`. Classes include fixed values on zoom (using `px`).
+
+{{< callout >}}
+**Using the CSS Grid layout module?** Consider using [the gap utility](#gap) instead.
+{{< /callout >}}
+
+### Notation for fixed values
+
+Spacing utilities that apply to all breakpoints, from `2xs` to `3xl`, have no breakpoint abbreviation in them. This is because those classes are applied from `min-width: 0` and up, and thus are not bound by a media query. The remaining breakpoints, however, do include a breakpoint abbreviation.
+
+The classes are named using the format `{property}{sides}-{size}` for `2xs` and `{property}{sides}-{breakpoint}-{size}` for `xs`, `sm`, `md`, `lg`, `xl`, `2xl`, and `3xl`.
+
+Where *property* is one of:
+
+- `m` - for classes that set `margin`
+- `p` - for classes that set `padding`
+
+Where *sides* is one of:
+
+- `t` - for classes that set `margin-top` or `padding-top`
+- `b` - for classes that set `margin-bottom` or `padding-bottom`
+- `s` - (start) for classes that set `margin-left` or `padding-left` in LTR, `margin-right` or `padding-right` in RTL
+- `e` - (end) for classes that set `margin-right` or `padding-right` in LTR, `margin-left` or `padding-left` in RTL
+- `x` - for classes that set both `*-left` and `*-right`
+- `y` - for classes that set both `*-top` and `*-bottom`
+- blank - for classes that set a `margin` or `padding` on all 4 sides of the element
+
+Where *size* is one of:
+
+- `none` - for classes that eliminate the `margin` or `padding` by setting it to `$ouds-dimension-0` aka `0`
+- `smash` - for classes that set the `margin` or `padding` to `$ouds-dimension-25` aka `2px`
```

Comment:
Thinking out loud here, but it might be hard to maintain. We could use maybe the `$ouds-dimension-base *` notion instead? Like it's done on Boootstrap with `$spacer *`. Not entirely sure it's better in terms of maintenance, let's keep that question in mind.

Reply:
Let's not change anything.